spec/requests/posts_specs.rb

```diff
@@ -3,8 +3,8 @@
 RSpec.describe Post, type: :model do
```

Comment:
- [ ] There are no test cases found in the post controller for `index` and `show` actions. Kindly ensure to add appropriate test cases for the post controller, and most importantly you should provide spec test for the `index` and `show` actions in the post controller.